src/RVRP_data.jl

```diff
@@ -50,37 +51,40 @@ mutable struct Request # can be
     price_reward::Float64 # define if semi_mandatory or optional; reward for fulfilling the request
     product_quantity::Float64 # of the request
     shipment_capacity_consumption::Vector{Float64} # can include several independant capacity consumptions: as weight, value, volume
-    shipment_measures::Vector{Float64} # can include several independant measures: as heigh, depth, width
-    pickup_point_or_group_id::String # empty string for delivery-only requests. id of the OperationPoint or of an OperationGroup representing alternatives for pickup
-    delivery_point_or_group_id::String # empty string for pickup-only requests. id of the OperationPoint or of a OperationGroup representing alternatives for delivery
+    incompatible_vehicles::Vector{{String,Int}} # list of Vehicle Category,  index of compartment
+    pickup_location_or_group_ids::Vector{String}  # empty string for delivery-only requests. id of the Locations or of LocationGroups representing alternatives for pickup
```

Comment:
Group definition are optional; making them mandatory can be very painful for the users when you have a specific group for each cause. So vector allow to use list of Locations; or a mix of both scheme

Reply:
In addition to offering different ways to encode the same information, I am afraid this will be a source of bugs when `pickup_availabitilies_at_location_or_group_ids` is not given in the same way as `pickup_location_or_group_ids`: When seeing vectors one may expect that one set of strings is the subset of the other.